.github/workflows/test.yml

```diff
@@ -211,7 +211,8 @@ jobs:
       - name: run pedantic clippy on workspace crates
         run: |
           cargo clippy --all-targets --all-features \
-          -- --warn clippy::pedantic --warn clippy::arithmetic-side-effects --deny warnings
+          -- --warn clippy::pedantic --warn clippy::arithmetic-side-effects \
+          --allow clippy::blocks_in_conditions --deny warnings
```

Comment:
I am working on updating to rust 1.81.0, but I am running into some issues: https://github.com/astriaorg/astria/pull/1523

Just a headsup so that we can remove it later.